src/cascadia/TerminalApp/App.cpp

```diff
@@ -631,11 +631,7 @@ namespace winrt::TerminalApp::implementation
 
     void App::_SetFocusedTabIndex(int tabIndex)
     {
-        auto tab = _tabs.at(tabIndex);
-        _tabView.Dispatcher().RunAsync(CoreDispatcherPriority::Normal, [tab, this](){

```

Comment:
You might need to add the Dispatcher.RunAsync(...) code back - I'm not sure this method is always called on the UI thread. 

For example, does pressing Alt+\<number\> to switch to a specific tab work with the change as it is currently?

Reply:
Uhh, yes it still works :P. Should I add it make nevertheless?

Reply:
... Probably yea. Might save someone else in the future :)

Reply:
Fixed

Reply:
Fixed